deploy/docker-entrypoint.sh

```diff
@@ -69,7 +69,7 @@ if [ "$1" = "nginx" ]; then
   # in this mode we're running in a separate container
   export APP_HOST=${APP_HOST:=app}
   exec_entrypoint "$ENTRYPOINT_PATH/nginx/"
-  exec nginx -c $OPT_DIR/nginx/nginx.conf -e /dev/stderr
```

Comment:
I got an issue with it in my containerisation saying that -e option is no longer supported by nginx.
At the same time, it's already in the conf.